src/util/quiz.js

```diff
@@ -15,14 +15,14 @@ class AnswerHandler {
 
   _setCurrentQuestion(cb) {
     var question = this.game.getCurrentQuestion();
-    var message = '<h1>' + question.questionString + '</h1>' +
-                  '<h3>Level: ' + this.game.level + '</h3>' + '<ol>';
+    var message = '<div style="line-height:100%; padding-top:10px; padding-bottom:10px; padding-left: 10px;padding-right:10px;">Level: ' + this.game.level + '' +
+    '<h3>' + question.questionString + '</h3>' + '<ol>';
```

Comment:
This won't apply to the 2d racer I think, only to wolf and the 3d racer. Is that what we intend here?